src/sentry/eventstore/models.py

```diff
@@ -681,6 +682,43 @@ def for_group(self, group: Group):
         return GroupEvent.from_event(self, group)
 
 
+@dataclass
+class IssueEvidence:
+    name: str
+    value: str
+    # Feels like we need more attributes to allow us to decide which things to include in verbose and
+    # condensed displays. We'll use all the evidence in issue details and emails, but only important
+    # rows in slack, for example. Some thoughts:
+    # important: bool
+    # include_in: Sequence[str] # List of places this should be included in, like UI, email, slack.
+    # Maybe too configurable? Also doesn't expand well as we add more integration types. Could be
+    # more general like: UI, alerts_email, alerts_messaging, integration_ticketing, etc.
```

Comment:
I feel like that should live in the integration instead of here

Reply:
maybe during the integration message builder? I guess I'd rather have each integration hardcoded (with a default base class method) than having to know up front which integrations the evidence would be appropriate it.

just thinking about this as a development model in order to customize a specific integration we'd just start adding the data during ingestion as opposed to designing/committing new code to the integration.

I guess thinking through all that I like the general model in that case

Reply:
Adding new code to the integrations for each new issue type doesn't scale well, although it wouldn't be too bad short term. Longer term, if we're accepting custom issues from 3rd parties then we won't be able to add in these customizations though.

I'm not sure if `include_in` really makes sense either though. But ideally we'd have some kind of metadata that the integrations can inspect to decide what to display, rather than customizing each time.

Reply:
I think the `important` (or whatever we want to call it) boolean would be sufficient, but idk how we'd enforce there ever only being one true.

Reply:
We could scope out a sort of generic set of building blocks (graph, detail, blame line, etc) and keep it open. Also a catchall `detail1, detail2, detailN` etc. could work.

Also possibly could do `key, value, value_short` for truncated versions

Reply:
> I think the `important` (or whatever we want to call it) boolean would be sufficient, but idk how we'd enforce there ever only being one true.

Would we require that there only actually be one? I wouldn't mind just allowing folks to set whatever they think is important and we push it into the message. If we only support one item for a particular integration we could just assume the first one (up to the vertical team to make sure it makes sense there)